core/controllers/editor_test.py

```diff
@@ -1713,7 +1723,7 @@ def test_get_with_invalid_version_raises_error(self):
         self.save_new_valid_exploration(exp_id, self.owner_id)
         self.get_json(
             '%s/%s' % (feconf.EXPLORATION_DATA_PREFIX, exp_id),
-            params={'v': 'invalid_version'}, expected_status_int=404)
+            params={'v': '546'}, expected_status_int=404)
```

Comment:
'546' is a random exploration version -- just to make sure it is a string convertible into int.